proposals/infra/INF-0004-register-types-and-diagnostics.md

```diff
@@ -474,7 +479,21 @@ if udt is set:
       emit err_hlsl_unsupported_register_type_and_variable_type  
 ```
 
-## Alternatives considered (Optional)
+## Behavioral Differences
+
+This infrastructure will introduce some behavioral differences between `clang-dxc` and `dxc`.

```

Comment:
I don't think you're intending to imply that there will be further differences between clang-dxc and clang? If we're describing the behavior in clang we should probably refer to it as clang. If we're referring to any behavior that's specific to when running through the clang-dxc interface then we should call that out then.

Reply:
I'm not sure I come to the same conclusion @damyanp. The line wants to distinguish behavioral differences between `clang-dxc` and  dxc. I don't see any mention of `clang`

Reply:
My point here is that clang-dxc is just the driver for running clang with the various flags set up so that the command-line looks like dxc.  The thing we're actually building (and what this document is describing) is how clang behaves.

Reply:
I agree, changed to `clang`